autorest/codegen/models/code_model.py

```diff
@@ -44,6 +48,9 @@ def get_serialization_type(self):
     def get_python_type(self, namespace=None):
         return self.type
 
+    def get_python_type_annotation(self) -> str:
+        return self.get_python_type()
```

Comment:
Need a namespace parameter, this will fail if called. You're just lucky we don't do type annotation in the client yet :)
I wonder how mypy was fine with that?

Reply:
still the case right?